src/serialize.js

```diff
@@ -86,6 +90,12 @@ const serializeResource = (type, resource, options, root = false) => {
     relationships: serializedRelationships
   } = serializeRelationships(relationships, relationshipOptions)
 
+  if (isPolymorphic) {
+    if (!attributes._type) throw `You did not specify a type for the relationship to the object with id "${identifier}"`
```

Comment:
Not sure about that, but it would give a hint on how to do that
```suggestion
    if (!attributes._type) throw `You did not specify a _type for the relationship to the object with id "${identifier}"`
```